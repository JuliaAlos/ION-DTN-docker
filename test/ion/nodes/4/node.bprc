1
a scheme ipn 'ipnfw' 'ipnadminep'
a endpoint ipn:4.0 x
a endpoint ipn:4.1 q
a endpoint ipn:4.2 x
a endpoint ipn:4.64 x
a endpoint ipn:4.65 x
a protocol udp 1400 100
a induct udp 192.168.0.104:4112 udpcli
a induct udp 192.168.0.104:4113 udpcli
a induct udp 192.168.0.104:4115 udpcli
a outduct udp 192.168.0.102:2114 udpclo
a outduct udp 192.168.0.103:3114 udpclo
a outduct udp 192.168.0.105:5114 udpclo
a plan ipn:1.0
a planduct ipn:2.0 udp 192.168.0.102:2114
a planduct ipn:3.0 udp 192.168.0.103:3114
a plan ipn:2.0
a planduct ipn:2.0 udp 192.168.0.102:2114
a plan ipn:3.0
a planduct ipn:3.0 udp 192.168.0.103:3114
a plan ipn:5.0
a planduct ipn:5.0 udp 192.168.0.105:5114 
s
