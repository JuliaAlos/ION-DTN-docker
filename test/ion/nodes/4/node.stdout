[i] admin pgm using SDR parm overrides from bench.ionconfig.
wmKey:           0
wmSize:          40000000
wmAddress:       0
sdrName:        ''
sdrWmSize:       0
configFlags:     1
heapWords:       20000000
heapKey:         -1
logSize:         0
logKey:          -1
pathName:       '/tmp'
Initializing semaphores to use: Posix Named Semaphores - max : 2048
Stopping ionadmin.
Stopping ionsecadmin.
Stopping ionadmin.
Stopping bpadmin.
